3-platform/warehouse/pom.xml

```diff
@@ -19,6 +19,7 @@
     <module>app</module>
     <module>domain</module>
     <module>spec</module>
+    <module>db_postgres</module>
```

Comment:
it is not maven module. so no need to have it here
```suggestion
```

Reply:
Maybe, that point means folder db/postgres is created.